back/assignment-1/src/main/java/ro/utcn/spet/example/a1/controller/QuestionController.java

```diff
@@ -0,0 +1,28 @@
+package ro.utcn.spet.example.a1.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import ro.utcn.spet.example.a1.dto.QuestionDTO;
+import ro.utcn.spet.example.a1.service.QtagManagementService;
+import ro.utcn.spet.example.a1.service.QuestionManagementServive;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class QuestionController {
+    private final QuestionManagementServive questionService;
+
+    @GetMapping("/questions")
+    public List<QuestionDTO> readAll() {
```

Comment:
You don't have methods / code for handling filtering request. This means that you are filtering the question on the UI... I would want to see it done on the backend.